dataprep/eda/distribution/render.py

```diff
@@ -1420,112 +1421,149 @@ def stats_viz_dt(stats: Dict[str, Any]) -> Dict[str, Dict[str, str]]:
     return {"Overview": {k: _format_values(k, v) for k, v in stats.items()}}
 
 
-def render_distribution_grid(
-    itmdt: Intermediate, yscale: str, plot_width: int, plot_height: int
-) -> Dict[str, Any]:
+def render_distribution_grid(itmdt: Intermediate, cfg: Config) -> Dict[str, Any]:
     """
-    Render plots and dataset stats from plot(df)
-    """  # pylint: disable=too-many-locals
-    figs: List[Figure] = list()
+    Create visualizations for plot(df)
+    """
+    # pylint: disable=too-many-locals
+    plot_width = cfg.plot.width if cfg.plot.width else 324
```

Comment:
What does this mean,`cfg.plot.width > 0`? In this case I'm not sure whether it works for negative number, would be better to write as `cfg.plot.width > 0`. The same comment also applies to other places like line 1489.

Reply:
cfg.plot.width is a global parameter, the default is None. But I think you are right, we can add one more constraint to keep it positive

Reply:
@jinglinpeng why do we need to check that this parameter is positive, but we don't need to check that other parameters are reasonable?

Reply:
@brandonlockhart  I'm not sure whether this line `cfg.plot.width if cfg.plot.width else 324` checks that the width is positive or checks the width is not None. For both cases it's better to write it explicitly.

Reply:
Ya this line `cfg.plot.width if cfg.plot.width else 324` does not check that the width is positive. And to be clear, as @dylanzxc said, the default of `cfg.plot.width` is None and it is only not None if the user specifies the value. But the user could set `cfg.plot.width` equal to a string (which would throw an error), so checking that the value is positive seems random to me. I think checking that the user specifies a valid parameter is a separate problem from this PR, and we maybe shouldn't do ad hoc checks throughout the code. Just my opinion.

Reply:
I've checked this in the new commit `    plot_width = cfg.plot.width if cfg.plot.width and cfg.plot.width > 0 else 972`. Should I revoke?